packages/backend-for-frontend/src/routers/clientRouter.ts

```diff
@@ -275,7 +275,7 @@ const clientRouter = (
           ctx
         );
 
-        return res.status(200).send(bffApi.PublicKeys.parse({ keys }));
+        return res.status(200).send(bffApi.PublicKeys.parse(keys));
```

Comment:
This is outside the scope the PR, but a minor correction: it was trying to parse an object instead of the array